editor/i18n/zh/components.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
             change_children: '连同修改子节点',
             change_self: '只修改节点自身',
         },
+        lightProbeGroup:{
+            generateTip: '在场景里重新生成探针',
```

Comment:
After pressing the Generate button, a dialog box needs to pop up to confirm overwriting the existing probes